### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by hmarchadour**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### How to test this PR?
+
+- [ ] JUnit
+- [ ] Cypress : please specify
+- [ ] Manual Test : please specify
+
+### Checklist
+
+- [ ] I have read CONTRIBUTING.md carefully.
```
Comment (pcdavid) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: The file actual file is simply `CONTRIBUTING` in our case.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+...
+ 
+
+### Potential side effects
+
+...
+
+### How to test this PR?
+
+- [ ] JUnit
```
Comment (pcdavid) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: More like "Unit tests" in general. JUnit for the backend parts, but jest for the front.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+### Type of this PR 
+
+- [ ] Bug fix
+- [ ] Improvement
+- [ ] Documentation
+- [ ] Cleanup
+- [ ] Test
```
Comment (pcdavid) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: - [ ] Build/Releng

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+### Type of this PR 
+
+- [ ] Bug fix
+- [ ] Improvement
```
Comment (pcdavid) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: "New feature or improvement"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- [ ] JUnit
+- [ ] Cypress : please specify
+- [ ] Manual Test : please specify
+
+### Checklist
+
+- [ ] I have read CONTRIBUTING.md carefully.
+- [ ] I have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
+- [ ] All my commits are signed-off (`-s`) with your mail address of your Eclipse Account.
```
Comment (pcdavid) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: "your" => "my" to be consistent. The other items are in the first person.

**Review by pcdavid — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (hmarchadour) [reply .github/PULL_REQUEST_TEMPLATE.md]: + I've split in
- [ ] Frontend Unit tests
- [ ] Backend Unit tests

**Review by pcdavid — APPROVED** [review]

**Pull request merged**